tests/functional/UseCase/Table/PreviewTableTest.php

```diff
@@ -0,0 +1,494 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\StorageDriver\FunctionalTests\UseCase\Table;
+
+use Google\Protobuf\Internal\GPBType;
+use Google\Protobuf\Internal\RepeatedField;
+use Google\Protobuf\NullValue;
+use Google\Protobuf\Value;
+use Keboola\Datatype\Definition\Teradata;
+use Keboola\StorageDriver\Command\Bucket\CreateBucketResponse;
+use Keboola\StorageDriver\Command\Table\CreateTableCommand;
+use Keboola\StorageDriver\Command\Table\DropTableCommand;
+use Keboola\StorageDriver\Command\Table\PreviewTableCommand;
+use Keboola\StorageDriver\Command\Table\PreviewTableResponse;
+use Keboola\StorageDriver\Credentials\GenericBackendCredentials;
+use Keboola\StorageDriver\FunctionalTests\BaseCase;
+use Keboola\StorageDriver\Teradata\Handler\Table\Create\CreateTableHandler;
+use Keboola\StorageDriver\Teradata\Handler\Table\Drop\DropTableHandler;
+use Keboola\StorageDriver\Teradata\Handler\Table\Preview\PreviewTableHandler;
+use Keboola\TableBackendUtils\Escaping\Teradata\TeradataQuote;
+
+class PreviewTableTest extends BaseCase
```

Comment:
asi se mi líbí co tady testuješ ale nelíbí se mi moc "jak".
- Prvně bych metody creat/drop/fill Table hodil do BaseCase, uz vidim jak se budou hodit jinde
- stejně tak asi repeatedStringToArray
- v metode `testCreateTable` (ten nazev je uz divnej) testujes vlastne tri usecasy. Asi bych to radeji videl aspon pres dataProvider a tu tabulku vytvaret v setUpu pokud neexistuje (nechat si nejake ID v instanci).
- nevidim zadne invalidni screnario

takze bych navrhoval malej refactoring.

Reply:
- `repeatedStringToArray` presunuta do `ProtobufHelper`
- prejmenovana metoda `testCreateTable`
- pridano invalidni scenario

Ostatni jsme se dohodli, ze nechame zatim jak je, a refaktorujeme pozdeji